.github/workflows/push-image.yaml

```diff
@@ -0,0 +1,73 @@
+---
+name: Build and push image to ECR
+
+on:
+  workflow_dispatch:
+    inputs:
+      docker_base_distribution:
+        description: "Enter the Docker Base Distribution (bionic, focal, stretch, buster, or bullseye)"
+        required: true
+        type: string
+        default: bullseye
+      network_name:
+        description: "Enter the Network Name (berkeley or mainnet)"
+        required: true
+        type: string
+        default: mainnet
```

Comment:
This one is a bit confusing to me. I think that whether we want to run it against berkley or mainnet is driven by the env variable `NETWORK` which doesn't seem to be set in the dockerfile. (Perhaps I'm missing something)